cura/NetworkClient.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2018 Ultimaker B.V.
```

Comment:
This class is not used anymore now that we bind directly to callbacks on the reply, so we can remove it.